modules/imported-account/providers.tf

```diff
@@ -0,0 +1,4 @@
+provider "aws" {
+  profile = var.profile
```

Comment:
We should use the standard `aws` provider in `providers.tf` so we can support both `profile` and explicit `role_arn` to assume, no?